pkg/hvutils/hvutils.go

```diff
@@ -55,8 +55,11 @@ func GetInstanceTypeFromTags(tags []string) (string, error) {
 		return "", ErrNoInstanceTypeFound
 	}
 	if len(instanceTypes) > 1 {
-		return "", fmt.Errorf("[GetInstanceTypeFromTags] more than one instance type. instanceTypes %v: %w",
-			instanceTypes, ErrMoreThanOneTagFound)
+		return "", fmt.Errorf(
```

Comment:
was this your formatter? You could have left it as it was. But this is also fine.

Reply:
I used `golines`.